src/Compilers/CSharp/Portable/Binder/ForEachLoopBinder.cs

```diff
@@ -768,8 +768,8 @@ private bool SatisfiesGetEnumeratorPattern(ref ForEachEnumeratorInfo.Builder bui
         {
             LookupResult lookupResult = LookupResult.GetInstance();
             MethodSymbol getEnumeratorMethod = FindPatternMethod(collectionExprType, GetEnumeratorMethodName, lookupResult,
-                                                                 _syntax.Expression, warningsOnly: true, diagnostics: diagnostics,
-                                                                 _syntax.SyntaxTree, MessageID.IDS_Collection);
+                                                     _syntax.Expression, searchExtensionMethods:false, warningsOnly: true, diagnostics: diagnostics,
```

Comment:
Parameters on newlines should either be indented once into the new line or aligned with the first character after the opening parenthesis.